### ISSUE TITLE
Why windows only?

### ISSUE THREAD

**Issue by bbecausereasonss** [YenRaven/annoy_ltm#1]
Any chance for a WSL/Ubuntu variant?

**Comment by YenRaven** [comment]
I have heard that it is possible it works on linux as is, but I do not have a way to test this myself. If you are interested in giving it a try, please report an issue for anything you run into. 
The reason I put that note into the repo is because annoy, the db used on the backend of this project is a cpp library that I chose specifically for its support of windows, which is what I'm running oobabooga on.

**Comment by Dalhimar** [comment]
During my initial installation of `annoy_ltm` I had an error that was related to `n_core_web_sm`. This is part of the spacy module. I was able to run `python -m spacy download en_core_web_sm` and after that it worked fine. I have successfully tested it with PygmalionAI 7B 4bit, and 13B 4bit models.. However I don't know how much my other module installs may or may not have helped. Since my Ubuntu install is only for playing with AI and on an isolated array, I haven't bothered with using separate Environments.
